website/docs/reference/dbt-jinja-functions/cross-database-macros.md

```diff
@@ -884,6 +887,35 @@ For databases that support it, this macro will return `NULL` when the cast fails
 
 ## Date and time functions
 
+### date
+
+**Availability**:
+dbt v1.8 or later. For more information, select the version from the documentation navigation menu.
+
+<VersionBlock firstVersion="1.8">
+
+__Args__:
+
+ * `year`: an integer
+ * `month`: an integer
+ * `day`: an integer
+
+This macro converts the `year`, `month`, and `day` into an SQL `DATE` type.
```

Comment:
```suggestion
This macro converts the `year`, `month`, and `day` into a SQL `DATE` type.
```

Reply:
hmmm, i was taught this -- non blocker so merging

![Screenshot 2024-05-09 at 1 42 08 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/d67bca2b-de14-43e6-8b43-94fdb18fcd01)